shared/timer/Timer.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Timer.h"
+
+Timer::Timer() : running(false) {}
```

Comment:
initialize startTime and endTime using default initializer:
startTime(), endTime()